seed_isort_config.py

```diff
@@ -18,10 +18,10 @@
     '.editorconfig', '.isort.cfg', 'setup.cfg', 'tox.ini', 'pyproject.toml',
 )
 THIRD_PARTY_RE = re.compile(
-    r'^known_third_party([ \t]*)=([ \t]*)(?:.*)?$', re.M,
+    r'^known_third_party([ \t]*)=([ \t]*)(?:.*?)?(\r?)$', re.M,
 )
 KNOWN_OTHER_RE = re.compile(
-    r'^known_((?!third_party)\w+)[ \t]*=[ \t]*(.*)$', re.M,
+    r'^known_((?!third_party)\w+)[ \t]*=[ \t]*(.*?)(\r?)$', re.M,
```

Comment:
this doesn't need to be captured here, this isn't used for replacing like `THIRD_PARTY_RE`

Reply:
I've put it here to be consistent with both regs. But you are right, with the `ini_load` and `toml_load` it's not necessary.